pages/components/shared/StyledChip.tsx

```diff
@@ -8,7 +8,7 @@ interface Props {
 
 const StyledChip: React.FC<Props> = ({ text, size, rounded }) => {
     return (
-        <div className={`bg-[#A1BFF9] px-3 py-2 font-bold text-${size} rounded-${rounded} my-2 xl:my-0`}>
+        <div className={`bg-[#A1BFF9] px-3 py-2 font-bold text-${size} rounded-${rounded} my-0`}>
```

Comment:
I don't think it makes sense to grow the tags vertically on smaller screens: they already can take up a lot of space (especially when the card width is small)

Reply:
That is the main issue. Should we show less tags or any other way around?

Reply:
![image](https://user-images.githubusercontent.com/3993495/194233520-20f52b69-08e4-465b-97d2-d41b4ede71d9.png)

do you think something like this will work?
The tags are limited to 1 line and ellipsis is used if there isn't enough space.